docs/notes/2.23.x.md

```diff
@@ -95,6 +94,8 @@ The `tfsec` linter now works on all supported platforms without extra config.
 
 #### Javascript
 
+The Node.js runtime now uses version 18.20.3 by default. This is a major version upgrade from the 16.x series, which was used before.
```

Comment:
Should mention that new pinned package manager versions?

Reply:
Added this in!